Hometask8/src/test/java/homework15/UITestsDemo.java

```diff
@@ -0,0 +1,101 @@
+package homework15;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class UITestsDemo {
+    WebDriver driver;
+
+    @BeforeMethod
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.get("https://www.bstackdemo.com/");
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        if (driver != null) {
+            driver.quit();
+        }
+    }
+
+    String quantityOfProducts = "2";
+    String product1Name = "iPhone 12 Pro Max";
+    String xpathExpression1 = String.format("//p[text()='%s']/following-sibling::div[@class='shelf-item__buy-btn']", product1Name);
+
+    double product1Price = 1099.00;
+    double product2Price = 649.00;
+    String product2Name = "iPhone XS Max";
+    String xpathExpression2 = String.format("//p[text()='%s']/following-sibling::div[@class='shelf-item__buy-btn']", product2Name);
+    String[] productsNames = {product1Name, product2Name};
+    double[] productsPrices = {product1Price, product2Price};
+
+    @Test
+    public void checkAddToCart() throws InterruptedException {
+        WebElement phone1 = driver.findElement(By.xpath(xpathExpression1));
+        Thread.sleep(1000);
+        phone1.click();
+
+        WebElement cartCloseBtn = driver.findElement(By.xpath("//div[@class='float-cart__close-btn']"));
```

Comment:
```suggestion
    WebElement cartCloseBtn = driver.findElement(By.className("float-cart__close-btn"));
```
Also, please, review similar cases.